index.js

```diff
@@ -0,0 +1,7 @@
+const include = __dirname;
+const isNodeApiBuiltin = process.versions.modules >= 52;
```

Comment:
Is there a story for versioning N-API or do we always just provide whichever version is provided by the napi header/cc?